adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -105,6 +105,8 @@ func CreateRoutesWithClusters(adapterInternalAPI model.AdapterInternalAPI, inter
 	apiVersion := adapterInternalAPI.GetVersion()
 
 	conf := config.ReadConfigs()
+
+	// Get the timeout from the default config as this is used for the enforcer timeout
```

Comment:
enforcer timeout? what does it mean? there's no usage for enforcer of a cluster timeout